embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/injection/DataCaptureServiceModuleImplTest.kt

```diff
@@ -63,7 +63,6 @@ internal class DataCaptureServiceModuleImplTest {
             FakeVersionChecker(false)
         )
 
-        assertTrue(module.powerSaveModeService is NoOpPowerSaveModeService)
```

Comment:
Might be worth replacing with `assertNotNull`

Reply:
sure!